recipes/py-linux-ports/meta.yaml

```
@@ -0,0 +1,39 @@
+{% set name = "py-linux-ports" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 2d7e6ecfe4458c9c087d414013ba2961a6a0541c5d441eea8d432d456a31ce01
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - PyLinuxPorts
```

```suggestion
test:
  imports:
    - PyLinuxPorts
  commands:
    - pip check
  requires:
    - pip
```
This just double checks that the correct versions are installed.